src/Plugin/tmgmt/Translator/MyGengoTranslator.php

```diff
@@ -198,7 +216,7 @@ public function getSupportedTargetLanguages(Translator $translator, $source_lang
    *   - Status object with order info.
    */
   public function sendJob(Job $job, $quote_only = FALSE) {
-    $data = tmgmt_flatten_data($job->getData());
+    $data = \Drupal::service('tmgmt.data')->filterTranslatable($job->getData());
```

Comment:
this might not actually be correct, despite what I said since it didn't use the array_filter() stuff before. We need to check this.